examples/petsc/bps.c

```diff
@@ -78,7 +78,7 @@ struct RunParams_ {
   MPI_Comm comm;
   PetscBool test_mode, read_mesh, user_l_nodes, write_solution;
   char *filename, *hostname;
-  PetscInt local_nodes, degree, q_extra, dim, num_comp_u, *m_elem;
+  PetscInt local_nodes, degree, q_extra, dim, num_comp_u, *mesh_elem;
```

Comment:
Part of my philosophy with these name changes is that the PETSc (and solid mechanics) examples seem to be the entry point for people looking to understand and use libCEED (especially for new grad students), so I'm ok with slightly more verbose names that help ease that learning curve.

Reply:
Very good point. This verbosity could have saved me a lot of time when I first started reading the fluids example.

Reply:
Some of the pain with the fluids example comes from the fact I was the one who started it and it was my first attempt at making an mini-app :) hopefully this wave of style + modularity refactors will make things easier for everyone down the road